src/modules/user/user.service.ts

```diff
@@ -29,8 +28,8 @@ export class UsersService {
   }
 
   async getByEmailAndPass(email: string, password: string) {
-    const passHash = createHmac('sha256', password).digest('hex');
-    return await this.userRepository.findOne({email, password: passHash});
+    // const passHash = createHmac('sha256', password).digest('hex');
```

Comment:
the password transform takes already care of that. Did work without the additional hashing.